admin/install_server.sh

```diff
@@ -21,6 +21,11 @@ echo "client_encoding = utf8" >> "$PG_CONF"
 
 service postgresql restart
 
+# Less compiler
+curl -sL https://deb.nodesource.com/setup | sudo bash -
```

Comment:
Is the node in our base image too old?
If it is, and we need to install the package ourselves, please do this step manually (copy list file to sources.list.d, import gpg key) instead of using curl|bash

or can we just install node-less directly? (I know this can be problematic because it's often quite old)